lib/util/import-file.js

```diff
@@ -4,8 +4,9 @@
  * @param {string} filepath
  * @todo WARN on version 10 and '--experimental-modules' and '--esm'
  */
+const isTypeModule = process.env.npm_package_type === 'module'
```

Comment:
JSDoc above it belong to the function, so it would make sense to move the constant a bit higher.